src/linear_algebra/bilinear_form.lean

```diff
@@ -948,39 +948,37 @@ end to_matrix
 
 end matrix
 
-namespace refl_bilin_form
-
-open refl_bilin_form bilin_form
+namespace bilin_form
 
 /-- The proposition that a bilinear form is reflexive -/
 def is_refl (B : bilin_form R M) : Prop := ∀ (x y : M), B x y = 0 → B y x = 0
 
-variable (H : is_refl B)
+namespace is_refl
+
+variable (H : B.is_refl)
 
 lemma eq_zero : ∀ {x y : M}, B x y = 0 → B y x = 0 := λ x y, H x y
 
 lemma ortho_sym {x y : M} :
   is_ortho B x y ↔ is_ortho B y x := ⟨eq_zero H, eq_zero H⟩
 
-end refl_bilin_form
-
-namespace sym_bilin_form
-
-open sym_bilin_form bilin_form
+end is_refl
 
 /-- The proposition that a bilinear form is symmetric -/
-def is_sym (B : bilin_form R M) : Prop := ∀ (x y : M), B x y = B y x
+def is_symm (B : bilin_form R M) : Prop := ∀ (x y : M), B x y = B y x
 
-variable (H : is_sym B)
+namespace is_symm
+
+variable (H : B.is_symm)
 
 lemma sym (x y : M) : B x y = B y x := H x y
 
-lemma is_refl : refl_bilin_form.is_refl B := λ x y H1, H x y ▸ H1
+lemma is_refl : B.is_refl := λ x y H1, H x y ▸ H1
 
 lemma ortho_sym {x y : M} :
-  is_ortho B x y ↔ is_ortho B y x := refl_bilin_form.ortho_sym (is_refl H)
+  is_ortho B x y ↔ is_ortho B y x := H.is_refl.ortho_sym
 
-lemma is_sym_iff_flip' [algebra R₂ R] : is_sym B ↔ flip_hom R₂ B = B :=
+lemma is_sym_iff_flip' [algebra R₂ R] : B.is_symm ↔ flip_hom R₂ B = B :=
```

Comment:
```suggestion
lemma is_symm_iff_flip' [algebra R₂ R] : B.is_symm ↔ flip_hom R₂ B = B :=
```

Reply:
ACtually, this should probably be in the `bilin_form` namespace, so that it's `bilin_form.is_symm_iff_flip'` not `bilin_form.is_symm.is_symm_iff_flip'.